Library/Formula/mongodb.rb

```diff
@@ -3,10 +3,10 @@
 class Mongodb < Formula
   homepage "https://www.mongodb.org/"
 
-  depends_on "go" => :build
```

Comment:
This is duplicated further below, also in the main body, hence the removal here.